File: PRAK402_2510817210004_MuhammadRizkyramadhani.c
#include <stdio.h>

int main() {
    int batas;

    printf("Selamat datang di program Ganjil & Genap!\n");
    printf("_________________________________________\n\n");

    printf("Masukkan batas bilangan : ");
    scanf("%d", &batas);

    printf("_________________________________________\n\n");

    printf("Berikut adalah hasilnya :\n");

    for (int i = 1; i <= batas; i++) {
        if (i % 2 == 1) {
            printf("%d ", i);
        }
    }
    printf("\n");

    for (int i = batas; i > 1; i--) {
        if (i % 2 == 0) {
            printf("%d ", i);
        }
    }

    return 0;
}

File: PRAK402_2510817210004_MuhammadRizkyramadhani.py
#Program Ganjil & Genap
print('Selamat datang di program Ganjil & Genap!')
print('_________________________________________\n')
batas = int(input("Masukkan batas bilangan : "))
print('_________________________________________\n')

print('Berikut adalah hasilnya :')
for i in range(1, batas + 1):
    if i % 2 == 1:
        print(i, end=' ')
print()
for i in range(batas, 1, -1):
    if i % 2 == 0:
        print(i, end=' ')